### PR TITLE
NAS-107651 / 12.0 / Nas 107651

### PR DESCRIPTION
**Pull request by erinix**

### CODE REVIEW

File: src/app/helptext/storage/volumes/datasets/dataset-form.ts [hunk 1]
```diff
@@ -126,6 +126,15 @@ dataset_form_default: T('Default'),
 
 dataset_form_quota_too_small: T("Quota size is too small, enter a value of 1 GiB or larger."),
 
+dataset_form_special_small_blocks_placeholder: T('Metadata (Special) Small Block Size'),
+dataset_form_special_small_blocks_tooltip: T('This value represents the threshold block size\
+ for including small file blocks into the special allocation class. Blocks smaller than or\
+ equal to this value will be assigned to the special allocation class while greater blocks\
+ will be assigned to the regular class. Valid values are zero or a power of two from 512B \
+ up to 1M. The default size is 0 which means no small file blocks will be allocated in the\
+ special class.  Before setting this property, a special class vdev must be added to the pool.\
```
Comment (Mrt134) [inline src/app/helptext/storage/volumes/datasets/dataset-form.ts:135]:
```suggestion
 special class. Before setting this property, a special class vdev must be added to the pool.\
```

File: src/app/helptext/storage/volumes/datasets/dataset-form.ts [hunk 2]
```diff
@@ -126,6 +126,15 @@ dataset_form_default: T('Default'),
...
+ See zpool(8) for more details on the special allocation'),
```
Comment (Mrt134) [inline src/app/helptext/storage/volumes/datasets/dataset-form.ts:136]: Added a link to ZFS on Linux manual page, but maybe a different link is better?
```suggestion
 See <a href="https://zfsonlinux.org/manpages/0.7.13/man8/zpool.8.html" target="_blank">zpool(8)</a> for more details on the special allocation'),
```

**Review by Mrt134 — APPROVED** [review]
Couple small things with the help text.

**Review by dkmullen — APPROVED** [review]

**Comment by william-gr** [comment]
@erinix  This was supposed to be merged to U1. Why was this merged now? ;)

**Pull request merged**